src/engine/bank01.asm

```diff
@@ -7172,7 +7172,45 @@ Func_6cfa: ; 6cfa (1:6cfa)
 	ret
 ; 0x6d3f
 
-	INCROM $6d3f, $6d84
+Func_6d3f: ; 6d3f (1:6d3f)
+	or a
+	bit 7, [hl]
+	ret z
+	push hl
+	bit 6, [hl]
+	ld a, $0a
+	ld hl, $26      ; copies from the byte of the damage due to poison by 10
+	jr z, .asm_6d52
```

Comment:
Now that we know this branching has to do with being poisoned versus being double poisoned, the label `.asm_6d52` can be appropriately renamed.